bin/run_tests

```diff
@@ -202,6 +202,7 @@ test_block_info() {
 
 test_battleship() {
   run_docker_test ./families/battleship/battleship_unit_tests.yaml
+  run_docker_test ./families/battleship/tests/test_tp_battleship.yaml
```

Comment:
What's the reasoning for removing them and then adding them back? Instead of just not removing them?

Reply:
It seems weird to me to have a commit that has tests that won't run due to missing code. Given that and the fact that there were changes to the tests, it felt easier to just re-introduce them and make those changes in a separate commit.

Reply:
I agree that it is weird that the commit has the test that will fail but it also has two downsides.  It removes the history from the original tests and makes it hard to see what the changes you made to the tests are. I would suggest not removing the tests to begin with.